source/edge-server/configure.txt

```diff
@@ -468,25 +398,31 @@ Upgrade Edge Server Procedure
 
    .. step:: Run the upgrade command
 
-      From the ``edge_server`` directory, run 
-      :ref:`the relevant upgrade command <edge-server-upgrade-commands>`
+      Run :ref:`the relevant upgrade command <edge-server-upgrade-commands>`
       to upgrade to your preferred version of the Edge Server:
 
       .. code-block:: shell
 
-         make upgrade-minor
+         edgectl update --minor
 
       Depending on your hardware and network connection, the upgrade 
       process may take seconds to minutes.
 
-   .. step:: Start the Edge Server
+   .. step:: Configure the Edge Server
+
+      After upgrading, ``edgectl`` prompts you to update the configuration.
+
+      .. code-block:: shell
+
+         edgectl config
+   
+   .. step:: Run the Edge Server to Deploy Changes
 
-      After upgrading, from the ``edge_server`` directory, run the command 
-      to start the Edge Server:
+      After upgrading, run the Edge Server to deploy the updates.
 
       .. code-block:: shell
 
-         make up
+         edgectl start
 
    .. step:: (Optional) Reset the Edge Server
```

Comment:
is this the only place we document how to reset the server?

Reply:
Yeah. I think we have generally only needed to document this in the context of upgrading the Edge Server version. Although if there are other scenarios where people need to reset the server, it may be worth breaking this out into a separate section. 🤔